arch/arm/boot/dts/overlays/README

```diff
@@ -3120,6 +3120,15 @@ Params: cma-512                 CMA is 512MB
                                 outputs)
 
 
+Name:   vc4-kms-vga666
+Info:   Enable the VGA666 (resistor ladder ADC) for the vc4-kms-v3d driver.
+        Requires vc4-kms-v3d to be loaded.
+Load:   dtoverlay=vc4-kms-vga666,<param>
+Params: ddc                    Enables GPIOs 0&1 as the I2C to read the EDID
```

Comment:
The indentation on the parameter description is off-by-one, but otherwise this looks OK.

Reply:
Fixed.